.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -1,11 +1,11 @@
 ---
 name: Bug Report
-description: Create a report to help us improve
+description: "🐞 Create a report to help us improve"
 body:
   - type: markdown
     attributes:
       value: |
-        Issues are for **concrete, actionable bugs and feature requests** only. For debugging help or technical support, please use:
+        Bug Report issues are for **concrete, actionable bugs** only. For debugging help or technical support, please use:
         - The #ansible-awx channel on irc.libera.chat
         - The awx project mailing list, https://groups.google.com/forum/#!forum/awx-project
```

We can put markdown in here.

```suggestion
        - The [awx:ansible.com](https://matrix.to/#/#awx:ansible.com) channel on Matrix
        - The `#ansible-awx` channel on irc.libera.chat
        - The [AWX project mailing list](https://groups.google.com/forum/#!forum/awx-project)
```

---

We might also want to consider trying to reference https://github.com/ansible/awx/#get-involved so that we have a single place for links to the mailing list/IRC/etc.